mbuild/compound.py

```diff
@@ -1238,12 +1238,12 @@ def save(self, filename, show_ports=False, forcefield_name=None,
             extensions are: 'hoomdxml', 'gsd', 'gro', 'top', 'lammps', 'lmp'
         show_ports : bool, optional, default=False
             Save ports contained within the compound.
-        forcefield_name : str, optional, default=None
+        forcefield_file : str, optional, default=None
             Apply a forcefield to the output file using a forcefield provided
             by the `foyer` package.
         forcefield_name : str, optional, default=None
-            Apply a forcefield to the output file using the `foyer` package and
-            a specific forcefield.xml file.
+            Apply a named forcefield to the output file using the `foyer`
+            package, ie. 'oplsaa'.
```

Comment:
Minor grammar: this should be `e.g.` 

Could you also add a link to this folder so people can easily see which force fields exist? https://github.com/mosdef-hub/foyer/tree/master/foyer/forcefields